src/mmw/sass/components/_modals.scss

```diff
@@ -138,3 +138,20 @@
   top: 1rem;
   left: 2rem;
 }
+
+.fa {
+  font-size: 1.0rem;
+  padding: 0 0.1rem 0 0.1rem;
+}
```

Comment:
Also the "Delineate Watershed" tab:

![screen shot 2016-10-05 at 2 47 35 pm](https://cloud.githubusercontent.com/assets/4165523/19126882/b37a78e0-8b0a-11e6-90b3-1a0cff2eac30.png)